react/fields/MonthPicker/MonthPicker.js

```diff
@@ -0,0 +1,175 @@
+import styles from './MonthPicker.less';
+
+import React, { Component, PropTypes } from 'react';
+import classnames from 'classnames';
+
+import ErrorIcon from '../../icons/ErrorIcon/ErrorIcon';
+import CustomMonthPicker from './CustomMonthPicker/CustomMonthPicker';
+import NativeMonthPicker from './NativeMonthPicker/NativeMonthPicker';
+
+function combineClassNames(props = {}, ...classNames) {
+  const { className, ...restProps } = props;
+
+  return {
+    className: classnames.apply(null, [...classNames, className]), // eslint-disable-line no-useless-call
+    ...restProps
+  };
+}
+
+export default class MonthPicker extends Component {
```

Comment:
🤔  Interesting. The only thing is there is a nice consistency between all the fields in style guide atm in the way they're set out. Personally I found this way more readable for a component as large as this (although not that complex). Value in both approaches to me. @mjt01 thoughts?